### DISCUSSION TITLE
Exceeding Database Size Limit

### DISCUSSION THREAD

**Discussion by rbeezley** [supabase/supabase#17563] [Questions]
I am using the free plan and my dashboard shows I have exceeded the database size limit.  

I added up the size of all my tables and it is less than 1 mb.

How can I find out what is consuming all the space and what can I do about?

I opened a ticket, but it says free plan is not guaranteed a response and to get help here. 

I appreciate any help...

![supabase1](https://github.com/supabase/supabase/assets/138222684/e4ea400c-e3d5-40ac-80eb-fdcf3bb92b3e)
![supabase2](https://github.com/supabase/supabase/assets/138222684/9ec095de-1e5f-4720-9135-9e0c3d8716d4)
![supabase3](https://github.com/supabase/supabase/assets/138222684/717d86b3-614f-4e11-9ad1-1571363e3b8f)

**Comment by avgupta456** [comment]
Also running into this issue.
